src/solvers/dg/3d/dg.jl

```diff
@@ -1276,6 +1285,9 @@ function rhs!(dg::Dg3D, t_stage)
   # Reset u_t
   @timeit timer() "reset ∂u/∂t" dg.elements.u_t .= 0
 
+  # Apply positivity preserving limiter of Zhang and Shu
+  @timeit timer() "PP limiter of Zhang&Shu" apply_positivity_preserving_limiter!(dg)
```

Comment:
```suggestion
  @timeit timer() "apply positivity-preserving limiter" apply_positivity_preserving_limiter!(dg)
```
The name is not relevant in the timer output and might be outdated if we switch to a different limiter or allow different kinds of limiters...